app/tests/spec/views/marketing_snippet.js

```diff
@@ -70,7 +71,7 @@ define(function (require, exports, module) {
         createView({
           language: 'en',
           service: 'sync',
```

Comment:
nit: There is a  `sync` service as a constant, https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/constants.js#L42